### PR TITLE
update lxd space test to test non-fan behavoir

### PR DESCRIPTION
**Pull request by nskaggs**
## Description of change

Update test for FAN changes to ensure we are testing old behavior. 

## QA steps
Check the lxd-container-space test.

### CODE REVIEW

File: acceptancetests/assess_lxd_space.py [hunk 1]
```diff
@@ -182,6 +182,10 @@ def assess_lxd_container_space(client):
     assert_initial_spaces(client)
     assert_initial_subnets(client)
     assert_added_space(client)
+    # With FAN now working, we need to explicitly set the model-config 
+    #to local to test the old behavior
```
Comment (wupeka) [inline acceptancetests/assess_lxd_space.py]: s/#to/# to/

**Comment by nskaggs** [comment]
$$merge$$

**Comment by nskaggs** [comment]
Result is of new run is:

ProvisioningError: ('0/lxd/0', 'unable to setup network: host machine "0" has no available FAN devices in space(s) "testspace"')

**Comment by nskaggs** [comment]
$$merge$$

**Pull request merged**